lib/organization.js

```diff
@@ -168,7 +181,7 @@ export class Organization {
     // throughout, but this assumption is not warranted
     const contracts = await getDeployedContracts();
     if (contracts.defaultVotingMachine) {
-      org.votingMachine = AbsoluteVote.at(contracts.defaultVotingMachine);
+      org.votingMachine = AbsoluteVote.at(contracts.defaultVotingMachine.address);
```

Comment:
Is the assumption here that AbsoluteVote is the default voting machine ?
where is that defined ?

Reply:
@orenyodfat Yes, altering that assumption will be part of a future issue.

Reply:
@dkent600 Could you please open an issue for that ?

Reply:
@orenyodfat Do you need it right now?  There are several issues that need to be created to bring arc-js in sync with new stuff that has been added to arc.  I would prefer to go over all of those at the same time and add the issues coherently, efficiently, completely and thoughtfully.

Would be good to discuss outside of this PR...

Reply:
I guess that open an issue is the good way to trace it...we might forgot that after this pr will be merged .

Reply:
No way am I going to forget that.  :-)  In my mind it is one of the highest priority things coming up.  I just want to be able to focus on it when I (or we) do it.

Reply:
@orenyodfat I went ahead and created this:  https://github.com/daostack/arc-js/issues/24

Reply:
resolved